doc/articles/getting-started/wizard/extensions.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: Uno.GettingStarted.UsingWizard.Extensions
+---
+
+Enables setting which Uno Extensions should be included with the generated projects.
+Uno Extensions is a set of Extensions that cover common essentials application core building blocks, and are supported on all operating systems Uno Platform targets.  
+The Uno.Extensions follows the Microsoft.Extensions model that creates a host environment where you can register additional dependencies. The registered dependencies are then available throughout the application via the Services (`IServiceProvider`) property on the `IHost` instance.
+    
+To learn more about Uno Extensions, visit [this page](xref:Overview.Features).
+    
+The available extensions are:
+    
+- **General**
+  - **Dependency Injection**
+    Dependency Injection (DI) is an Inversion of Control (IoC) technique which enables consuming classes not to worry about the management or creation of their dependencies, and instead rely on a central service to provide it when needed, based how the provider was configured.      
+    Uno Platform brings the power of [Microsoft Extensions Dependency Injection](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection) to all OSs Uno Platform targets, and adds its additional functionality via its Uno.Extensions.DependencyInjection library, which you can learn more about [here](xref:Overview.DependencyInjection).
+
+  - **Configuration**  
+    This extension provides a way to load application configuration data from and to various sources using the [Options Pattern](https://learn.microsoft.com/en-us/dotnet/core/extensions/options).
+    Refer to the [Uno Configuration](xref:Overview.Configuration) documentation for more information.
+
+  - **HTTP**
+    Uno.Extensions.Http allows for the registration of API endpoints as multiple typed HttpClient instances. In this centralized location for accessing web resources, the lifecycle of the corresponding HttpMessageHandler objects is managed. Added clients can optionally be configured to use the platform-native handler. Additional functionality is provided to clear cookies or log diagnostic messages in responses. This library uses Microsoft.Extensions.Http for any HTTP related work.
+    For more documentation on HTTP requests, [read the documentation](xref:Overview.Http).
+
+    Another related package installed with the default template is Refit. This is a port of the [Refit library](https://github.com/reactiveui/refit) that enables mapping REST APIs into .NET interfaces.
```

Comment:
The package is installed when selecting Http, not specific to the default template